app/routes/agenda/press.js

```diff
@@ -0,0 +1,87 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class AgendaPressRoute extends Route {
+  @service store;
+
+  async model() {
+    const { meeting, agenda } = this.modelFor('agenda');
+    const agendaitems = await this.store.query('agendaitem', {
+      filter: {
+        agenda: {
+          id: agenda.id,
+        },
+        type: {
+          ':uri:': CONSTANTS.AGENDA_ITEM_TYPES.NOTA,
+        },
+        treatment: {
+          'decision-activity': {
+            'decision-result-code': {
+              ':uri:': CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD,
+            },
+            subcase: {
+              confidential: false,
+            }
+          },
+        },
+        ':or:': {
+          'is-approval': false,
+          ':has-no:is-approval': true,
+        }
+      },
+      include: 'mandatees,agenda-activity.subcase.type',
+    });
+
+    return {
+      meeting,
+      agendaitems: await this.sortAgendaitems(agendaitems),
+    };
+  }
+
+  async sortAgendaitems(agendaitems) {
```

Comment:
Re-implementation of [the code from the `agenda-submission` service](https://github.com/kanselarij-vlaanderen/agenda-submission-service/blob/main/lib/agendaitem-order.js#L91-L130).